ckanext/gla/search.py

```diff
@@ -34,3 +41,36 @@ def debug(context, data_dict={}):
         return conn.search(**params).__dict__
     except Exception as e:
         raise common.SearchError(e.args)
+
+logfile = "/srv/app/search_logs.csv"
+
+def _result_index(page, index_in_page):
+    page_idx = 0 if _empty_or_none(page) else int(page) - 1
+    return page_idx * 20 + int(index_in_page) + 1
```

Comment:
The number of datasets per page can be set in the [config](https://github.com/ckan/ckan/blob/master/ckan/config/config_declaration.yaml#L1078), so it might be best to link this to that config value in case it get's changed at some point?

Also, according to the docs `loop.index` (in `package_list.html`) starts at 1 already:
![image](https://github.com/GreaterLondonAuthority/dfl-ckanext/assets/1217533/52dff114-7fe4-4dca-b2fc-4b350648f60c)
so I'm pretty sure we don't need the extra `+1` here

```suggestion
    from ckan.common import config
    return page_idx * config.get('ckan.datasets_per_page') + int(index_in_page)
```